src/content/structured/components/checkboxes/code.mdx

```diff
@@ -140,66 +144,79 @@ export const snippetsHelperText = [
-</IcCheckboxGroup>
-<IcCheckboxGroup label="This is a default checkbox group" name="2">
-  <IcCheckbox value="valueName1" label="Unselected / Default" />
-  <IcCheckbox value="valueName2" label="Selected / Default" checked />
+    snippet: `<IcCheckboxGroup label="Make changes to your order" name="1" size="small">
+  <IcCheckbox value="valueName1" label="Extra shot (50p)" />
+  <IcCheckbox value="valueName2" label="Soya milk" checked />
 </IcCheckboxGroup>
-<IcCheckboxGroup label="This is a large checkbox group" name="3" size="large">
-  <IcCheckbox value="valueName1" label="Unselected / Large" />
-  <IcCheckbox value="valueName2" label="Selected / Large" checked />
+<IcCheckboxGroup label="Make changes to your order" name="2">
+  <IcCheckbox value="valueName1" label="Extra shot (50p)" />
+  <IcCheckbox value="valueName2" label="Soya milk" checked />
 </IcCheckboxGroup>
-<IcCheckboxGroup label="Checkboxes define size in this checkbox group" name="4">
-  <IcCheckbox value="valueName1" label="Small" size="small" />
-  <IcCheckbox value="valueName2" label="Default" />
-  <IcCheckbox value="valueName3" label="Large" size="large" />
+<IcCheckboxGroup label="Make changes to your order" name="3" size="large">
+  <IcCheckbox value="valueName1" label="Extra shot (50p)" />
+  <IcCheckbox value="valueName2" label="Soya milk" checked />
 </IcCheckboxGroup>`,
   },
 ];
 
 <ComponentPreview snippets={snippetsSizes} style={{ gap: "24px" }}>
-  <IcCheckboxGroup label="This is a small checkbox group" name="1" size="small">
-    <IcCheckbox value="valueName1" label="Unselected / Small" />
-    <IcCheckbox value="valueName2" label="Selected / Small" checked />
+  <IcCheckboxGroup label="Make changes to your order" name="1" size="small">
+    <IcCheckbox value="valueName1" label="Extra shot (50p)" />
+    <IcCheckbox value="valueName2" label="Soya milk" checked />
   </IcCheckboxGroup>
-  <IcCheckboxGroup label="This is a default checkbox group" name="2">
-    <IcCheckbox value="valueName1" label="Unselected / Default" />
-    <IcCheckbox value="valueName2" label="Selected / Default" checked />
+  <IcCheckboxGroup label="Make changes to your order" name="2">
+    <IcCheckbox value="valueName1" label="Extra shot (50p)" />
+    <IcCheckbox value="valueName2" label="Soya milk" checked />
   </IcCheckboxGroup>
-  <IcCheckboxGroup label="This is a large checkbox group" name="3" size="large">
-    <IcCheckbox value="valueName1" label="Unselected / Large" />
-    <IcCheckbox value="valueName2" label="Selected / Large" checked />
+  <IcCheckboxGroup label="Make changes to your order" name="3" size="large">
+    <IcCheckbox value="valueName1" label="Extra shot (50p)" />
+    <IcCheckbox value="valueName2" label="Soya milk" checked />
   </IcCheckboxGroup>
-  <IcCheckboxGroup
-    label="Checkboxes define size in this checkbox group"
-    name="4"
-  >
+</ComponentPreview>
+
+### Setting each checkbox size 
```

Comment:
We should never show different sized checkboxes in the same group together. Remvoe this whole section.

Reply:
I think we need to show how to add size directly to the checkbox and not just the checkbox group. Is there a way this could be displayed better? Maybe copying the sizes example above with three different checkbox groups but set the size on the checkboxes instead of the group? Or have an example without a checkbox group, so just three single checkboxes with the different sizes...

Reply:
@DBD324 we have implemented the latter. Let me know if you want me to change this.